contributing/code/bc.rst

```diff
@@ -291,6 +301,8 @@ Change value of a constant                          Yes [1]_ [5]_
        Changing an argument type is only possible with a parent type.
        Changing a return type is only possible with a child type.
 
+.. [9] Allowed for the ``void`` return type.
```

Comment:
Are you saying that `void` is an exception then? That, for example, adding/removing a return type would be a BC break... *except* for void?

Reply:
Yes, that's what I mean. Technically, PHP allows removing the return type without breaking children. But doing so would allow breaking contract. Except for void.